packages/edit-site/src/components/header-edit-mode/index.js

```diff
@@ -41,11 +42,7 @@ const preventDefault = ( event ) => {
 	event.preventDefault();
 };
 
-export default function Header( {
-	openEntitiesSavedStates,
-	isEntitiesSavedStatesOpen,
-	showIconLabels,
```

Comment:
These are all props that don't need to be passed top-down, instead we can retrieve them in useSelect, that's the only changes here. I can definitely extract this to its own PR but it feels too small to be worth it. (Won't really have any impact on the size of this PR)